.github/workflows/build.yaml

```diff
@@ -101,7 +101,7 @@ jobs:
     steps:
       - uses: actions/checkout@v4
       - name: Download font artefact files
-        uses: actions/download-artifact@v4
+        uses: actions/download-artifact@v5
```

Comment:
I'll fold that with other tweaks, to actually give this a test run from the PR workflow.